.github/workflows/label_commenter_config.yml

```
@@ -0,0 +1,9 @@
+comment:  # Leaving these empty keys here in case we want to add to them in the future
+  header: ''
+  footer: ''
+
+labels:
+  - name: help wanted
+    labeled:
+      issue:
+        body: 👋 Thank you for your suggestion or request! While the EUI team agrees that it's valid, it's unlikely that we will prioritize this issue on our roadmap. We'll leave the issue open if you or anyone else in the community wants to own it or implement it by contributing to EUI. If not, this issue will auto close in a year.
```

I would love feedback from the team as to:

- Copy/wording
- Any other labels we can think of that we would want to auto-respond to (maybe something like "Hi! The EUI team will be working on this immediately" or "Hi! The EUI team will be working on this in the next 6 months to 1 year"?...)
- Any other use cases or feedback I haven't thought of

---

I've got just two small suggestions:

* What about omitting "to own it" in the sentence "...if you or anyone else in the community wants to own it or implement it by contributing..." I feel that would emphasize the community aspect of applying the change/fix.
* How do you feel about changing " a " to " one " in the last sentence so it says will close in one year.

---

The only other label I could see an auto-responder for might be `tech debt`. Not entirely sure what it'd say, but maybe something along the lines of 

```
👋 Thank you for your suggestion! The EUI team will review this item and try to prioritize it in the next one to two quarters.  Please contribute a pull request to EUI if you or anyone else in the community wants to own it or implement it
```

---

Love the copy suggestions, thanks Trevor! Don't be afraid to suggest them using GH's suggestion syntax for easier committing :)
```suggestion
labels:
  - name: help wanted
    labeled:
      issue:
        body: 👋 Thank you for your suggestion or request! While the EUI team agrees that it's valid, it's unlikely that we will prioritize this issue on our roadmap. We'll leave the issue open if you or anyone else in the community wants to implement it by contributing to EUI. If not, this issue will auto close in one year.
```

---

> The only other label I could see an auto-responder for might be tech debt.

My hesitation to that suggestion is that we apply the `tech debt` label to a lot of our own issues and it doesn't really make sense to have a bot telling us what to do with our own work 😅 For that reason, I think I lean towards making these auto responses either on an `assign:` label (which I want to discuss potentially overhauling at our next backlog meeting) or for labels intentionally meant for community/open source contributors.